systemtest/src/test/java/io/strimzi/systemtest/ConnectS2IST.java

```diff
@@ -332,6 +335,91 @@ void testJvmAndResources() {
         KafkaConnectS2IResource.deleteKafkaConnectS2IWithoutWait(kafkaConnectS2i);
     }
 
+    @Test
+    void testKafkaConnectorWithConnectS2IAndConnectWithSameName() {
+        String topicName = "test-topic-" + new Random().nextInt(Integer.MAX_VALUE);
+        String connectClusterName = "connect-cluster";
+        String connectS2IClusterName = "connect-s2i-cluster";
+
+        KafkaResource.kafkaEphemeral(CLUSTER_NAME, 3).done();
+        // Crate connect cluster with default connect image
```

Comment:
You have the comments from the oposite case - here is now S2I and on line 376 is regular connect.